canonical-web-and-design/ubuntu.com
# Simplify the Ubuntu Core on Raspberry Pi download pages

## Done

- moves to a single page for the pi 2, 3 and 4 on Ubuntu Core 18

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/iot
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- [x] compare to the copy doc [/download/iot index page](https://docs.google.com/document/d/1Sxdl5cGGe1wbHmKB2eL2zQLfVW3KBea02a2FIGx3VdQ/edit) and resolve comments
- [x] compare to the copy doc [/download/raspberry-pi-2-3-core page](https://docs.google.com/document/d/1Ig906jbgB39QHw61uHMvIsRbSBUjchJBDkMZl0WzlWM/edit?ts=5e626419#heading=h.1vxoguo2s9q5) and resolve comments
- [x] test the redirects /download/raspberry-pi-2-3-core and /download/raspberry-pi-compute-module-3 to the new /download/raspberry-pi-core page
- [x] compare to the copy doc [download mega menu](https://docs.google.com/document/d/11pX5gxOE6UWljIGRBZdwuCL1MHzs4pNgxJpK0mcIqdQ/edit#)

## Issue / Card

Fixes #6986
